realm/realm-library/src/main/java/io/realm/internal/Collection.java

```diff
@@ -367,7 +367,7 @@ public Collection(SharedRealm sharedRealm, LinkView linkView, SortDescriptor sor
 
         this.sharedRealm = sharedRealm;
         this.context = sharedRealm.context;
-        this.table = linkView.getTable();
+        this.table = linkView.getTargetTable();
```

Comment:
This constructor seems to be used not only in snapshot but in constructors of `RealmList`. Is this change valid for `RealmList` as well?